web/satellite/vuetify-poc/src/views/AccountSettings.vue

```diff
@@ -174,6 +200,13 @@ const userSettings = computed((): UserSettings => {
     return usersStore.state.settings as UserSettings;
 });
 
+/**
+ * Returns user's paid tier status from store.
+ */
+ const isPaidTier = computed<boolean>(() => {
+    return usersStore.state.user.paidTier ?? false;
```

Comment:
nit - I think `return usersStore.state.user.paidTier;` is sufficient. That is what we are doing elsewhere (e.g. components/LimitWarningBanners.vue)